src/App.css

```diff
@@ -1,38 +1,80 @@
-.App {
-  text-align: center;
+a {
+  text-decoration: none;
+  color: black;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.app-header,
+.footer {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #2b4570;
+  color: white;
+}
+
+/* .footer {
+  margin-top: auto;
+} */
```

Comment:
- [ ] Please, kindly remove this old piece of commented out code, in order to improve the cleanliness of the repo 💪